src/main/resources/com/google/api/codegen/go/doc.snip

```diff
@@ -15,5 +15,5 @@
     @end
     package {@view.fileHeader.localPackageName} // import "{@view.importPath}"
 
-    const gapicNameVersion = "gapic/0.1.0"
+    const gapicNameVersion = "gapic/20170208"
```

Comment:
This definition should be moved somewhere that will be easy to remember to update whenever a new GAPIC release is cut.